modules/sievefilters/modules.php

```diff
@@ -953,7 +954,7 @@ public function process() {
             if ($action->action == 'autoreply') {
                 $filter->addRequirement('vacation');
```

Comment:
We can now remove all $filter->addRequirement as they are added dynamically in each action